src/bootstrap/configure.py

```diff
@@ -44,10 +44,14 @@ def v(*args):
 o("verbose-tests", "rust.verbose-tests", "enable verbose output when running tests")
 o(
     "ccache",
-    "llvm.ccache",
-    "invoke gcc/clang via ccache to reuse object files between builds",
+    "build.ccache",
+    "invoke gcc/clang/rustc via ccache to reuse object files between builds",
```

Comment:
This is a potential breaking change for some people. I would duplicate it with a FIXME note instead of removing the old one.

Reply:
How could this be breaking? The outside interface is the same (`--enable-ccache`), it will just now set `build.ccache` instead of `llvm.ccache`, and `build.ccache` has a higher priority.

Reply:
Ah, I thought this was accepting llvm.ccache from CLI. Didn't check the entire code.